lists2safebrowsing.py

```diff
@@ -117,9 +206,11 @@ def find_hosts(disconnect_json, allow_list, chunk, output_file, log_file,
                 log_file.write("[hash] %s\n" % hashlib.sha256(canon_d).hexdigest());
               publishing += 1
               domain_dict[canon_d] = 1;
+              # TODO?: hashdata_bytes += hashdata.digest_size
```

Comment:
Do it.  Some abstraction to remove the hardcoded use of sha256 can be done but at this point it's probably not worth it since the client might be even more restricted than the service.